cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramNotificationSubscriberService.java

```diff
@@ -402,6 +402,19 @@ private void handleProgramEvent(ProgramRunId programRunId, ProgramRunStatus prog
         recordedRunRecord = appMetadataStore.recordProgramResumed(programRunId, messageIdBytes, resumeTime);
         writeToHeartBeatTable(recordedRunRecord, resumeTime, programHeartbeatTable);
         break;
+      case STOPPING:
+        Map<String, String> notificationProperties = notification.getProperties();
+        long stoppingTsSecs = getTimeSeconds(notificationProperties, ProgramOptionConstants.STOPPING_TIME);
```

Comment:
May be call it stoppingTime similar to other methods

Reply:
Terence and I were discussing that ideally timestamp should be in millis. Since this timestamp is in seconds we will explicitly suffix it with secs even if it means it deviates from the current naming convention